Makefile

```diff
@@ -0,0 +1,12 @@
+build: 
```

Comment:
let's change build -> sense 
so the command is ``` make sense ```

Reply:
Ahh! I remember this :laughing: